locale/en/manageiq.po

```diff
@@ -52249,22 +52070,25 @@ msgid "OVF Templates"
 msgstr ""
 
 #: ../app/javascript/components/schedule-form/schedule-form-fields.js
-#: ../app/views/miq_ae_tools/_results_tabs.html.haml
 msgid "Object"
 msgstr ""
 
 #: ../app/helpers/settings_schedule_helper.rb
+#: ../app/helpers/shared_helper/ab_show_helper.rb
 #: ../app/views/layouts/_ae_resolve_options.html.haml
-#: ../app/views/shared/buttons/_ab_show.html.haml
 #: ../app/views/static/ae_resolve_options/ae-resolve-options.html.haml
 msgid "Object Attribute"
 msgstr ""
 
-#: ../app/views/miq_ae_tools/_results_uri.html.haml
+#: ../app/views/layouts/_ae_resolve_options.html.haml
+msgid "Object Attribute 1"
+msgstr ""
```

Comment:
Commented in https://github.com/ManageIQ/manageiq-ui-classic/pull/9013 about this